sbin/common/config_init.sh

```diff
@@ -418,6 +425,28 @@ function setOpenjdkSourceDir() {
   fi
 }
 
+# Set custom image base if set
+function setCustomImage() {
+  local imageCandidate="${1}"
+  if [[ -z ${imageCandidate} ]] ; then
+    echo "default image will be used: ${BUILD_CONFIG[CONTAINER_IMAGE]}" 
+    return 1
+  fi
+  # is next param switch?
```

Comment:
I think you're trying to say "Is the imageCandidate being supplied via the switch parameter?" is that correct?

Reply:
Hmm... no?-) If the `imageCandidate` is empty (first if -z) then it is nogo.
If the `imageCandidate`  starts with `-` then it is switch. Thats the second condition after `# is next param switch?`  then again no go. Last is check whether it is main argumetn (jdk8u...). If so, again no go, otherwise it is the optional argument and will be used. 

I had added more comments to make it a bit more clearer.  But maybe I missused word "switch"